build.gradle.kts

```diff
@@ -99,7 +99,7 @@ allprojects {
             dependsOn("cleanTest")
         }
         withType<JavaCompile> {
-            options.compilerArgs.addAll(arrayOf("-Xlint:all", "-Werror", "-Xlint:-classfile", "-Xlint:-processing"))
+            options.compilerArgs.addAll(arrayOf("-Xlint:all", "-Werror", "-Xlint:-classfile", "-Xlint:-processing", "-Xlint:-try"))
```

Comment:
This allows us to write:

```
try(ISentryLifecycleToken ignored = scopes.makeCurrent()) {
	...
}
```

and rely on `AutoClosable` to clean up scopes.

Reply:
It complains about not using the variable (called `ignored` here) inside the `try {}` block.